### PR TITLE
Update Install.md

### PR DESCRIPTION
**Pull request by keileg**
## Proposed changes
Removed reference to 'master' branch from Install.md.

## Types of changes

What types of changes does this PR introduce to PorePy?
_Put an `x` in the boxes that apply._

- [ ] Minor change (e.g., dependency bumps, broken links).
- [ ] Bugfix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Testing (contribution related to testing of existing or new functionality).
- [ ] Documentation (contribution related to adding, improving, or fixing documentation).
- [ ] Maintenance (e.g., improve logic and performance, remove obsolete code).
- [x] Other: Install instructions

## Checklist

_Put an `x` in the boxes that apply or explain briefly why the box is not relevant._

Install instructions only.

- [ ] The documentation is up-to-date.
- [ ] Static typing is included in the update.
- [ ] This PR does not duplicate existing functionality.
- [ ] The update is covered by the test suite (including tests added in the PR).
- [ ] If new skipped tests have been introduced in this PR, `pytest` was run with the `--run-skipped` flag.

### CODE REVIEW

File: Install.md [hunk 1]
```diff
@@ -22,13 +22,8 @@ or for editable installs into the user directory:
...
     pip install --user -e .
 
-
-Installation of PorePy itself should be straightforward, following the instructions in Readme.md.
-
-To get the code fully working requires a few more steps, as described below.
-
 ## Installation on Windows
-Installation on Windows is currently (Spring 2021) rather easy, following the above instructions. The dependencies should be installed  using either `conda` or the Intel python distribution and then `pip install porepy` from source. 
+Installation on Windows is currently (Spring 2021) rather easy, following the above instructions. The dependencies should be installed using either `conda` or the Intel python distribution and then `pip install porepy` from source.
```
Comment (IvarStefansson) [inline Install.md:26]: Should we update from "Spring 2021"?

**Review by IvarStefansson — APPROVED** [review]

File: Install.md (hunk 1 above)
Comment (keileg) [reply Install.md:26]: If we have verified more recently that this still works, I would say. What do you think?

Comment (IvarStefansson) [reply Install.md:26]: I haven't done it in a while

Comment (keileg) [reply Install.md:26]: Me neither (not since spring 2021). Should we leave it then, but perhaps change to 'at the time of writing'

Comment (keileg) [reply Install.md:26]: Though that change by itself would falsify the modified statement

Comment (IvarStefansson) [reply Install.md:26]: Leave as is for now and perhaps check at some point and update

**Pull request merged**